ibm_mq/datadog_checks/ibm_mq/data/conf.yaml.example

```diff
@@ -85,6 +85,11 @@ instances:
     #
     # auto_discover_queues: false
 
+    ## @param mqcd_version - integer - optional - default: 6
+    ## Which API version to use
```

Comment:
Can we mention the valid versions we support? I noticed the ConfigurationError message says it must be a number between 1 and 9.

The docs here mentioned up to version 11, it might be useful the explain why we're limiting it to version 9.

Reply:
Pymqi only defines up to 9: https://github.com/dsuch/pymqi/blob/master/code/pymqi/CMQC.py#L2108